### ISSUE TITLE
The size of Select does not affect the size of SelectItem

### ISSUE THREAD

**Issue by cangSDARM** [akveo/react-native-ui-kitten#1417]
### To Reproduce

[select doc](https://akveo.github.io/react-native-ui-kitten/docs/components/select/overview#select)
![image](https://user-images.githubusercontent.com/20441896/116370755-4b9d3f00-a83d-11eb-9fc6-c73af14f5d0a.png)

### Expected behavior

The size of `Select` should directly affect the size of its sub-components, that is, `SelectItem`, so that it will appear more coordinated

### Link to runnable example or repository (highly encouraged)

[select doc](https://akveo.github.io/react-native-ui-kitten/docs/components/select/overview#select), see the part of "size"

## UI Kitten and Eva version

| Package      | Version |
| ----------- | ----------- |
| @eva-design/eva      |    latest    |
| @ui-kitten/components   |     latest    |

**Comment by cangSDARM** [comment]
Should it be changed to *enhancement*? I don’t know, but it’s not very coordinated, especially on small cellphones.

**Comment by Applizone** [comment]
Agree. Even I set the size of the ```<Select size='small' />```, the ```<SelectItem />``` dropdown listing is still on default size.
